### ISSUE TITLE
There was an error running the selected code generator: 'The RPC server is unavailable' when trying to generate an MVC Controller with views using EF

### ISSUE THREAD

**Issue by ianq22** [dotnet/Scaffolding#1697]
Howdy devs,

I've been attempting to learn C#/.NET since it seemed like a good fit for a project I'm planning on developing. While running through [part 4 of the official Microsoft tutorial ](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/adding-model?view=aspnetcore-5.0&tabs=visual-studio) I continually keep hitting a bug with the 'Add MVC Controller with views, using Entity Framework' scaffolding feature. 

I fill out the information and select my model, create the new data context type, and use the pre-generated Controller name. I make sure to leave the Generate views, Reference script libraries, and Use a layout page boxes checked.

Without fail, no matter how many times I run a clean install on Visual Studio, I encounter the same error stating that "There was an error running the selected code generator: 'The RPC server is unavailable'". Seeing as I've run through thistutorial several times and keep encountering the same thing, I'm starting to think it's not just me missing something that might be causing this.

Appreciate any help or guidance you folks might have. Hopefully I'm in the right spot for this!

Here's the version numbers of the software I'm running
Microsoft Visual Studio Community 2019
Version 16.11.5
VisualStudio.16.Release/16.11.5+31729.503
Microsoft .NET Framework
Version 4.8.04084

Installed Version: Community

.NET Core Debugging with WSL   1.0
.NET Core Debugging with WSL

ASP.NET and Web Tools 2019   16.11.75.64347
ASP.NET and Web Tools 2019

ASP.NET Web Frameworks and Tools 2019   16.11.75.64347

C# Tools   3.11.0-4.21403.6+ae1fff344d46976624e68ae17164e0607ab68b10
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Microsoft Visual Studio Tools for Containers   1.2
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

NuGet Package Manager   5.11.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit https://docs.nuget.org/

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

Razor (ASP.NET Core)   16.1.0.2122504+13c05c96ea6bdbe550bd88b0bf6cdddf8cde1725
Provides languages services for ASP.NET Core Razor.

SQL Server Data Tools   16.0.62107.28140

Visual Studio Code Debug Adapter Host Package   1.0

Visual Studio Container Tools Extensions   1.0

Visual Studio Tools for Containers   1.0

**Comment by Rick-Anderson** [comment]
@ianq22  can you install the .NET 6 SDK and try again?

**Comment by ianq22** [comment]
Hey @Rick-Anderson

So I ended up completely reinstalling Win 10 since the network drivers got so corrupted that I couldn't even load up the Network settings, it would just crash every single time despite my PC being connected to the internet. I'm guessing this is what messed things up during file generation as I ran through again after a clean install and was able to run the scaffolding generator no problem. Anyways thanks for checking in, appreciate the help!
